src/style.css

```diff
@@ -1,16 +1,26 @@
+body {
+  background-image: url(../img/wallpapersden.com_black-wall-pattern_2000x1338.jpg);
+  background-size: cover;
+  background-repeat: no-repeat;
+  color: #fff;
+  background-attachment: fixed;
+}
+
 h1 {
   font-family: cursive;
+  text-align: center;
```

Comment:
- [ ] Please let's follow the layout of the wireframe by adjusting the "Leaderboard" heading to the left rather than the center as shown in the image below 👇 
### Wireframe
![image](https://user-images.githubusercontent.com/46839548/186715499-f040bd34-128c-487e-9d72-71acf45df45a.png)
### Your Design
![image](https://user-images.githubusercontent.com/46839548/186715696-25305a80-b1a1-4f43-b8c6-9be371d56a9d.png)